graphrag/index/init_content.py

```diff
@@ -177,6 +181,28 @@
   # map_max_tokens: {defs.GLOBAL_SEARCH_MAP_MAX_TOKENS}
   # reduce_max_tokens: {defs.GLOBAL_SEARCH_REDUCE_MAX_TOKENS}
   # concurrency: {defs.GLOBAL_SEARCH_CONCURRENCY}
+
+drift_search:
```

Comment:
Most of these values have already been fine tuned, so not super sure if we want to expose them (given how big the config is now). Maybe something for discussion.

Reply:
Will discuss with broader config convo